lib/url.js

```diff
@@ -0,0 +1,4 @@
+import {URL} from 'url';
```

Comment:
`URL` has been a global in nodejs since v10, so I don't think there is a need for the browser/node variants here.

See history:
https://nodejs.org/api/url.html#url_the_whatwg_url_api